src/options.js

```diff
@@ -152,6 +152,7 @@ const DEFAULT_KEYBINDINGS = {
   showImagesLarge: ['z l'],
   showImagesMedium: ['z e'],
   showImagesIcon: ['z i'],
+  copyUrlKey: ['c', 'y'],
```

Comment:
Can you please remove the default value for the keybinding?
Going forward, I want to avoid adding new default keybindings. You can still configure it for yourself in the settings

Reply:
done.